src/main/java/httpserver/HTTPRequest.java

```diff
@@ -25,509 +25,502 @@
  * @see HTTPHandler
  */
 public class HTTPRequest implements Runnable {
```

Comment:
I'm sure they have done it better as well. This is probably a good idea at some point.